pkg/archive/changes.go

```diff
@@ -121,6 +122,9 @@ func isENOTDIR(err error) bool {
 	if err == nil {
 		return false
 	}
+	if err == syscall.ENOTDIR || err == unix.ENOTDIR {
```

Comment:
are there cases where `syscall.ENOTDIR != unix.ENOTDIR`?

Reply:
See the commit message, I've had problems with syscall.* and unix.* diverging in the past. But somehow this doesn't build in your CI, so I'll just go ahead and drop it :)